### ISSUE TITLE
Neofetch isn't displaying screen resolution on Cygwin

### ISSUE THREAD

**Issue by dar-irl** [dylanaraps/neofetch#242]
## Description
Not sure if I'm an idiot and just missing a dependency, but I can't seem to figure out why Neofetch won't display my screen resolution on Cygwin.

## System Information
Cygwin/Mintty

#### Neofetch version
Latest Neofetch from git

#### Operating system
Windows 10 Pro

## Verbose log
http://pastebin.com/raw/YY8jrXRd

**Comment by dylanaraps** [comment]
On windows neofetch uses these builtin commands to get the screen resolution. Try them in a CYGWIN terminal and let me know if they print anything.

```sh
wmic desktopmonitor get screenwidth /value
wmic desktopmonitor get screenheight /value
```

Does resolution work in ScreenFetch for you?

**Comment by dar-irl** [comment]
Strange, it's not returning any values for screenheight/width. I'm on a fresh install of Windows 10, perhaps I'm missing something that I had on my previous drive.

**Comment by dylanaraps** [comment]
Try out screenfetch and let me know if it displays the resolution. iirc screenfetch also uses wmic so if it also doesn't work then this might be an issue both scripts share.

**Comment by dar-irl** [comment]
Same issue, screenfetch doesn't display the resolution.

**Comment by dylanaraps** [comment]
The wmic command should work outside of the box and shouldn't need any extra dependencies. I'll open an issue on Screenfetch's GitHub to see if we can come up with a solution.

**Comment by dar-irl** [comment]
This seems to be a bug in Windows 10. I ran Windows 7 in a VM and the wmic command works fine.

**Comment by dar-irl** [comment]
`cmd /c wmic path Win32_VideoController get CurrentHorizontalResolution,CurrentVerticalResolution` within Cygwin seems to be a working alternative in Windows 10. Would this be difficult to implement?

**Comment by dylanaraps** [comment]
Fixed by #254

**Comment by brianscottwilson** [comment]
I just tried the "wmic desktopmonitor get screenheight /value" commands on my Win 10 piece of ... and oddly enough it did run.  So did the "wmic path Win32_VideoController get CurrentHorizontalResolution" command.  Looks like this has been fixed.  Now I'm looking for a working instance to install in Cygwin.
